__tests__/main.test.ts

```diff
@@ -19,14 +20,16 @@ describe("main", () => {
       listDifferent: false
     });
 
+    const expectedDirname = slash(__dirname);
```

Comment:
Was this necessary to run tests on windows?

Reply:
Yes.
Unrelated to issue, but actually the "logs nothing and does not exit if all files are valid" test still fails on windows. It has to do with \r \n symbols at the end of files, but i didn't have time to look into this further